app/views/admin/schedules/index.html.haml

```diff
@@ -25,8 +25,21 @@
                 method: :get, class: 'btn btn-primary'
                 = link_to 'Delete', admin_conference_schedule_path(@conference.short_title, schedule.id),
                 method: :delete, class: 'btn btn-danger', data: { confirm: "Do you really want to delete Schedule #{schedule.id}?" }
-
 .row
-  .col-md-12.text-right
+  .col-md-12
+- if @venue && @venue.rooms.any?
```

Comment:
Really sorry, was in a hurry to get my first issue done. Would take proper care of indentation from now on.